features/other_b/support.feature

```diff
@@ -21,39 +21,27 @@ Feature: Filing a support request
   When I follow "Support and Feedback"
     And I fill in "Brief summary" with "you suck"
     And I fill in "Your comment" with "blah blah blah"
-    And I fill in "Your email (optional)" with ""
+    And I fill in "Your email (required)" with "test@archiveofourown.org"
     And I select "Deutsch" from "feedback_language"
     And all emails have been delivered
     And I press "Send"
   Then I should see "Your message was sent to the Archive team - thank you!"
     And 1 email should be delivered
```

Comment:
This needs to be updated to two now that email addresses are required, but otherwise, this looks good to me!